plugins/gatsby-plugin-auto-index-pages/gatsby-node.js

```diff
@@ -0,0 +1,126 @@
+const path = require('path');
+const fromList = require('./utils/unist-fs-util-from-list');
+const visit = require('unist-util-visit');
+const generateHTML = require('./utils/generate-html');
+const { sentenceCase } = require('./utils/string');
+
+exports.createPages = async ({ actions, graphql, reporter }, pluginOptions) => {
+  const { skippedDirectories } = pluginOptions;
+  const { createPage } = actions;
+
+  const { data, errors } = await graphql(`
+    query MyQuery {
+      tableOfContents: allFile(
+        filter: {
+          sourceInstanceName: { eq: "markdown-pages" }
+          base: { in: ["index.mdx", "index.md"] }
+          relativePath: { regex: "/^(?!whats-new).*$/" }
```

Comment:
Should we be adding in all of the `skippedDirectories` to this regex statement?

Reply:
Ah, I see...you're skipping them in code down below.

Reply:
Oops, I meant to remove this regex and instead use the `skippedDirectories` only. Good catch! I will update this.

Reply:
Addressed in 00b6fc5